### PR TITLE
Adding JSON / Animation info for Reach

### PR DESCRIPTION
**Pull request by ILoveAGoodCrisp**

### CODE REVIEW

File: src/content/hr/engine/animation-data/readme.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+The [hr-ek] uses several types of animations when building a model_animation_graph from source data. Using legacy commands these are filetypes, however in the Sidecar pipeline animation types are defined in the sidecar.xml
```
Comment (num0005) [inline src/content/hr/engine/animation-data/readme.md:1]: is that a valid way to add a link?

**Review by num0005 — COMMENTED** [review]
at a glance that looks fine

File: src/content/hr/engine/animation-data/readme.md (hunk 1 above)
Comment (ILoveAGoodCrisp) [reply src/content/hr/engine/animation-data/readme.md]: Link works but it should have a more descriptive text link. Added. Also removed the workflow section of the page.yml, it was left over from the general animation-data page.

File: src/content/hr/guides/json-parameters/readme.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Below is a list of the possible JSON parameters that can be included within the JSON files generated during the [FBX to GR2][hr-tool#fbx-to-gr2] process. Note that in order for any changes to JSON files to take effect, you'll need to save your changes and re-run `fbx-to-gr2`.
+
+NOTE: When manually editing a JSON, all properties should feature the prefix `bungie_`
```
Comment (num0005) [inline src/content/hr/guides/json-parameters/readme.md:3]: change this into an alert and it's all good

**Review by num0005 — APPROVED** [review]

File: src/content/hr/guides/json-parameters/readme.md (hunk 2 above)
Comment (ILoveAGoodCrisp) [reply src/content/hr/guides/json-parameters/readme.md:4]: Added alert formatting

**Comment by num0005** [comment]
Sorry for the delay

**Pull request merged**